gnovm/memfile.go

```diff
@@ -87,6 +89,25 @@ func (mempkg *MemPackage) Validate() error {
 		prev = file.Name
 	}
 
+	pIndex := strings.Index(mempkg.Path, "/p/")
+	if pIndex > 0 && strings.Count(mempkg.Path[:pIndex], "/") == 0 {
+		for _, file := range mempkg.Files {
+			fset := token.NewFileSet()
+			astFile, err := parser.ParseFile(fset, file.Name, file.Body, parser.ImportsOnly)
+			if err != nil {
+				continue // can be other files like LICENSE, README or empty gno files
```

Comment:
If you apply previous suggestion:

Since we spent the resources parsing the file, I think it's better to invalidate the package in case of parsing error

```suggestion
				return fmt.Errorf("failed to parse imports in file %q of package %q: %w", mempkg.Path, file.Name, err)
```

Reply:
since empty gno files make `parser.Parsefile` return error, that would mean forbid empty gno files or i would have to check before if the file is empty to skip it too.

Reply:
cf90c37